### PR TITLE
Deprecate the kitbook

### PR DESCRIPTION
**Pull request by RealOrangeOne**
https://github.com/srobo/kitbook/commit/2bffe3f44405e31997d6b6679572d932a336c971

### CODE REVIEW

**Review by trickeydan — APPROVED** [review]

**Comment by sedders123** [comment]
On second thoughts, should this redirect to the runbook, so any links in the wild still work?

**Comment by PeterJCLaw** [comment]
> On second thought, should this redirect to the runbook so any links in the wild still work?

That would presumably depend on the same sub-page existing in the runbook? A placeholder page which handles `/kitbook*` and tells people about the move might be an alternative?

**Comment by sedders123** [comment]
I don't feel super strongly about it was just a thought, happy to go with consensus. Redirecting any kitbook route to the homepage of runbook feels like a low effort win. Given the relatively short lifespan of the kitbook I can't imagine there are too many links in the wild but the redirect would mean someone ends up at a semi useful page rather than a 404.

**Review by raccube — APPROVED** [review]

File: _env/nginx.conf [hunk 1]
```diff
@@ -160,6 +155,9 @@ http {
 
     # The runbook has replaced trac
     rewrite ^/trac                 /runbook/ permanent;
+    
+    # The kitbook has been incorperated back into the runbook
```
Comment (sedders123) [inline _env/nginx.conf:159]:
```suggestion
    # The kitbook has been incorporated back into the runbook
```
```suggestion
    # The kitbook has been incorperated back into the runbook
```

**Pull request merged**